src/main/java/com/origamisoftware/teach/agile/junit/SubscriptionPeriod.java

```diff
@@ -61,12 +66,17 @@ public int getTotalMonths() {
      * subscription period or false otherwise.
      *
      * @param date a date to consider
-     * @return true if true if the date comes after the expiration date of this
+     * @return true if the date comes after the expiration date of this
      * subscription period or false otherwise.
      */
     public boolean hasExpired(Date date) {
-        // todo implement method
-        throw new NotImplementedException();
+        // This is implemented
```

Comment:
these comments serve no purpose so they can be removed. 
also, single line comments begin with // while multi line comments use /* */